pages/common/man.md

```diff
@@ -18,6 +19,10 @@
 
 `man -w {{command}}`
 
+- Display the man page for a custom overrided locale (L):
+
+`man {{command}} -L {{locale}}`
```

Comment:
We prefer long argument format when is available:
```suggestion
`man {{command}} --locale={{locale}}`
```
(you can edit the other examples BTW)

Reply:
Should we have users perform this sort of update when they're just translating? @sbrl

Reply:
:eyes: this isn't a translation PR. I didn't get it.

Reply:
Nevermind, I didn't even read this PR's title, I just assumed it was a translation 😂

Reply:
@bl-ue lol, I was just thinking may be I could also give other contribution than translation ^^
@navarroaxel : Ok for the long version of the options, I was not aware of the convention, my mistake

Reply:
np @patricedenis, it's not explicit in the contribution guidelines, I think.